daemon/cmd/policy.go

```diff
@@ -244,6 +244,9 @@ func (d *Daemon) PolicyAdd(rules policyAPI.Rules, opts *policy.AddOptions) (newR
 // was not able to be imported.
 func (d *Daemon) policyAdd(sourceRules policyAPI.Rules, opts *policy.AddOptions, resChan chan interface{}) {
 	policyAddStartTime := time.Now()
+	if !opts.ProcessingStartTime.IsZero() {
```

Comment:
Is there still cases where we need to merge these?

Reply:
The only places I can see are K8s network policies ([pkg/k8s/watchers/network_policy.go](https://github.com/cilium/cilium/blob/master/pkg/k8s/watchers/network_policy.go#L90)) or test cases.